src/common/url/search-params.ts

```diff
@@ -19,3 +19,9 @@ export const createSearchParam = (params: Record<string, string>): string => {
   });
   return urlParams.toString();
 };
+
+export const removeSearchParam = (param: string): string => {
+  const urlParams = new URLSearchParams();
```

Comment:
You create a new empty `URLSearchParams`, and then remove a param from it 😄

Reply:
Yeah, noticed that a few seconds ago as well above 🤕.